src/react/components/SaleorProvider.tsx

```diff
@@ -1,34 +1,23 @@
-import React, { Dispatch, SetStateAction } from "react";
-import { Core, SaleorSDK, SaleorSDKConfig } from "../../core";
-import { ApolloClient, NormalizedCacheObject } from "@apollo/client";
+import React from "react";
+import { Core } from "../../core";
 
 export type SaleorContextType = {
```

Comment:
At the moment I believe `Core` is our entire context, so we can use:

```
export const SaleorContext = React.createContext<Core>(
  null
);
```

Reply:
I changed this, but it's `<Core | null>`